src/styles/main.scss

```diff
@@ -1 +1,67 @@
-// put code here
+body {
+  position: relative;
+  min-height: 100vh;
+  margin: 0;
+  padding: 0;
+}
+
+.stopwatch {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 80vmin;
+  height: 80vmin;
+  border: 10px dotted #000;
+  border-radius: 50%;
+
+  &__center {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    background-color: black;
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
+  }
+}
+
+.hand {
```

Comment:
if it's a separate block it should be moved to a separate file
or make it an element of stopwatch block (use modificators if you need)